packages/core/framework/src/http/middlewares/error-handler.ts

```diff
@@ -20,7 +20,9 @@ export function errorHandler() {
     res: Response,
     _: NextFunction
   ) {
-    const logger = req.scope.resolve(ContainerRegistrationKeys.LOGGER)
+    const logger = req.scope
```

Comment:
question: When is `req.scope` not defined?

Reply:
That's the mystery 😅 It should be defined all the time and we are not able to reproduce it locally.

So thought of using `console.log` to get access to the original error and find if there is something weird making `req.scope` not exist.

Reply:
Yeah, we found it is an issue with the DynamoDB connection which was initialized before we initialize the container. Therefore, the `req.scope` was undefined.